### PR TITLE
Basis for hooded clothes

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
Adds the framework for any further hooded clothing. 
Also adds my totally snowflake hooded jacket to the loadout. (Should be sprite consistent)

## Why It's Good For The Game
Hooded clothing! Style points, and atmosphere. Could also add hooded church clothing in the future, which would be a pretty cool flavor for Mehkane.

## Changelog
```changelog
add: Added hooded jacket code, and the sleeveless jacket item into loadout.
```

### CODE REVIEW

File: zzzz_modular_occulus/code/modules/client/preference_setup/loadout/lists/suits.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Suit slot
+/datum/gear/suit
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/modules/client/preference_setup/loadout/lists/suits.dm:2]: This bit doesn't need to be copypasta'd! You can comment it out if you wanna keep it for future reference without looking for the original file.

File: zzzz_modular_occulus/code/modules/clothing/suit/hooded.dm [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+				hood.canremove = FALSE
+				icon_state = "[toggleicon]_t"
+				H.update_inv_wear_suit()
+	else
+		RemoveHood()
+
+/* For anyone adding hooded clothes, take note.
+You need 2 versions of the sprite, one with the normal icon and one with icon name and '_t' at the end of it,
+for both Item and mob sprited versions.   EX: sleeveless_red & sleeveless_red_t
+Else, it won't work properly.**//
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/modules/clothing/suit/hooded.dm:65]: You added one extra * and /!

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Some quick tweaks here and there and it should work out fine!

File: zzzz_modular_occulus/code/modules/clothing/suit/hooded.dm (hunk 2 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/modules/clothing/suit/hooded.dm]: Fixed this!

File: zzzz_modular_occulus/code/modules/client/preference_setup/loadout/lists/suits.dm (hunk 1 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/modules/client/preference_setup/loadout/lists/suits.dm:2]: ...I got entirely confused on how the load out system works, and I copy pasted it without thinking. Whoops.

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks good!

**Comment by Michiyamenotehifunana** [comment]
Aseprite_xq5put2hRq 
Sprites for reference

**Comment by RadiantFlash7** [comment]
For reference, The hoods will block ears and anything extending past the head while the hood is up.

**Pull request merged**